lib/AST/ASTContext.cpp

```diff
@@ -6395,6 +6397,9 @@ LoadedExecutablePlugin *ASTContext::loadExecutablePlugin(StringRef path) {
 }
 
 void *ASTContext::loadLibraryPlugin(StringRef path) {
+  // Remember the path (even if it fails to load.)
+  getImpl().LoadedPluginLibraryPaths.insert(path);
```

Comment:
Do we want to track _failed_ libraries? If not, I can just move this after `getPluginRegistry()->loadLibraryPlugin(resolvedPath)` .

Also, do we want the VFS path or the "real" path here? I guess it doesn't matter because we only use this to get the module name from the filename. cc: @rxwei